### PR TITLE
DHFPROD-7876: Default views for first-time user experience

### PR DESCRIPTION
**Pull request by briantang**
### Description

- Reopening Akshay's PR after a couple conflict resolution and test fixes.

#### Checklist: 
```diff
- Note: do not change the below
```

-  ##### Owner:

- [x] JIRA_ID included in all the commit messages
- [x] PR title is in the format JIRA_ID:Title
- [x] Rebase the branch with upstream
- [x] Squashed all commits into a single commit
- [x] Code passes ESLint tests
- [x] Added Tests  

- ##### Reviewer:

- [x] Reviewed Tests
- [x] Added to Release Wiki

### CODE REVIEW

**Review by xnikhil08 — DISMISSED** [review]
The spinner keeps running when there are no entity instances in Final and the toggle is automatically switched to Staging (where it can't fins any entity instances for some reason)

STR:-
1. Load reference-entity-model project.
2. Delete everything in Final database.(from query console)
3. Login to HC and navigate to Explore tile.
4. The All entities option is selected, with Staging Database, which is good, but there is no data being rendered in UI from Staging and the spinner keeps running.

**Comment by briantang** [comment]
This was working properly until the PR here got merged: https://github.com/marklogic/marklogic-data-hub/pull/6552. Now a 500 Internal Server Error appears. @prolonMarkLogic could you take a look and see if you have any thoughts?

File: marklogic-data-hub-central/ui/e2e/cypress/integration/explore/entitiesSidepanel.spec.tsx [hunk 1]
```diff
@@ -30,12 +30,12 @@ describe("Test '/Explore' left sidebar", () => {
     Cypress.Cookies.preserveOnce("HubCentralSession");
     cy.restoreLocalStorage().then(() => {
       cy.log(`**Go to Explore section**`);
```
Comment (aniamy) [inline marklogic-data-hub-central/ui/e2e/cypress/integration/explore/entitiesSidepanel.spec.tsx:32]: Here the then can be removed.
And the log line added to the it below, next to the vist.

File: marklogic-data-hub-central/ui/e2e/cypress/integration/explore/entitiesSidepanel.spec.tsx [hunk 2]
```diff
@@ -144,6 +148,8 @@ describe("Test '/Explore' left sidebar", () => {
     entitiesSidebar.selectBaseEntityOption("Order");
     entitiesSidebar.getBaseEntityOption("Order").should("be.visible");
     cy.wait(1000);
+    browsePage.getStagingDatabaseButton().click();
```
Comment (aniamy) [inline marklogic-data-hub-central/ui/e2e/cypress/integration/explore/entitiesSidepanel.spec.tsx:151]: Why are we clicking on both buttons?

File: marklogic-data-hub-central/ui/e2e/cypress/integration/explore/entitiesSidepanel.spec.tsx (hunk 1 above)
Comment (briantang) [reply marklogic-data-hub-central/ui/e2e/cypress/integration/explore/entitiesSidepanel.spec.tsx]: Good point, removed.

File: marklogic-data-hub-central/ui/e2e/cypress/integration/explore/entitiesSidepanel.spec.tsx (hunk 2 above)
Comment (briantang) [reply marklogic-data-hub-central/ui/e2e/cypress/integration/explore/entitiesSidepanel.spec.tsx]: Definitely weird. Didn't see that before. Removed the first click.

**Review by aniamy — DISMISSED** [review]

**Comment by briantang** [comment]
Rebased to fix latest conflict.

**Review by xnikhil08 — APPROVED** [review]

**Review by Sanjeevani19 — DISMISSED** [review]

**Review by aniamy — APPROVED** [review]

**Review by yubiselmarklogic — APPROVED** [review]

**Pull request merged**